src/server/rdb_save.cc

```diff
@@ -733,15 +733,10 @@ io::Bytes RdbSerializer::PrepareFlush() {
   return mem_buf_.InputBuffer();
 }
 
-error_code RdbSerializer::WriteJournalEntry(const journal::Entry& entry) {
-  io::BufSink buf_sink{&journal_mem_buf_};
-  JournalWriter writer{&buf_sink};
-  writer.Write(entry);
```

Comment:
Is this not breaking compatibility?

Reply:
No, the serialized data is the same, we just do the serialization in journal_slice.cc (when I figure out what's the problem with the linking order)